models/Category.php

```diff
@@ -104,6 +104,14 @@ public static function getMenuTypeInfo(string $type): array
     {
         $result = [];
 
+        if (!class_exists(Theme::class)) {
+            return $result;
+        }
+
+        if (!class_exists(CmsPage::class)) {
+            return $result;
+        }
```

Comment:
@AdrienAdB you can probably drop the second class existence checks - if the `Theme` class is there, it's pretty much a given that the `CmsPage` class will also be there.
```suggestion
```